lib/OpenQA/Worker/WebUIConnection.pm

```diff
@@ -276,6 +276,15 @@ sub disable {
     $self->finish_websocket_connection;
 }
 
+# define list of HTTP error codes which indicate that the web UI is overloaded or down for maintenance
+# (in these cases the re-try delay should be increased)
+my %busy_error_codes = (502 => 1, 503 => 1, 504 => 1, 598 => 1);
```

Comment:
I think `my %BUSY_ERROR_CODES = map { $_ => 1 } 502, 503, 504, 598;` looks a bit better.

Reply:
(All upper case for package scope.)